### DISCUSSION TITLE
json field type support in sqlite?

### DISCUSSION THREAD

**Discussion by m5nv** [prisma/prisma#3575] [Q&A] [answered]
SQLite has supported JSON field type for a very long time now. I had looked prisma a year ago and it lacked JSON field type and so decided to stick with Sequelizer. With 2.x release I decided to give prisma another look after I read that it added support for JSON. The excitement has been short lived.

![image](https://user-images.githubusercontent.com/45653160/92643393-52ebf880-f2ff-11ea-985f-e0ef81fa44ac.png)

So apparently JSON field type support is not there for SQLite. Question time:
1. Is SQLite JSON field type support in your roadmap?
2. If so, any ball park estimate?
3. If not is there any good documentation to get one started in adding support?

Thanks!

**Comment by ryands17** [comment, marked as answer]
Hey @m5nv 👋 
I suppose you're talking about the [JSON extension](https://www.sqlite.org/json1.html) that SQLite supports. I don't think there's a feature request for that yet, so do go ahead an create one [here](https://github.com/prisma/prisma/issues/new/choose) explaining your use case as well :)

**Reply by m5nv** [reply]
Done!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
